src/doc/gui_manual/Contributing/index.rst

```diff
@@ -96,7 +96,6 @@ use of Sphinx as we move forward. These are discussed at the
   Note that the files listed in the ``.. toctree::`` block do not include
   their ``.rst`` extensions.
 
-* Wherever possible, keep lines in ``.rst`` files to 80 columns or less.
```

Comment:
Should there be a bullet here that says something to the effect that lines should consist of a single sentence regardless of length.

Reply:
Yup. Fixed.